### DISCUSSION TITLE
Improvements to Control Panel Terminology For End User

### DISCUSSION THREAD

**Discussion by tobyjug** [craftcms/cms#14602] [Ideas]
I recall it being mentioned a few times before about the possibility of renaming 'Entries' to 'Content' in the Control Panel — is this still the plan? The term 'Entry' is far too alien to the end-user and often causes friction during client handovers. Most users think in terms of pages and content, not Entries, Channels, Singles etc. I'd love to see some improvements in this area come to V5, so that the user doesn't have to concern themselves with the technicalities of how content is working behind the scenes.

Even better, allowing us to embed or pin channel/structure data to a single (or page as it should probably be referred to) so that they can navigate to a page in the CP, edit any content on that single, as well as the content from a related channel would be a huge boost in terms of UX.

A simple, but good example, would be a Meet The Team page with some copy at the top. At the moment, I would create a single where they can edit the copy and maybe some SEO related content, but then they'd have to navigate to the Team Members Channel to edit the Team entries. Whereas from the end-user's perspective, they just see a Meet The Team page on the front-end and think it should all be editable from the same place; and I tend to agree with them.

**Comment by thupsi** [comment]
The renaming of "Entries" to "Content" was/is part of a wider refactoring called "Unified Content View" that was initialy slated for Craft 5. It turned out to be too large of an undertaking for this version (especialy taking into account the allready big architectural changes with Matrix, DB etc) and has been pushed back to Craft 6.

That being said, there is widespread concensus both in the community and (AFAIK) within P&T themselves that there's room for improvement when it comes to making the whole content organization in the CP more intuitive for authors (hence the UCV plan). I don't think it's simply a matter of terminology, but also of content hierarchy, grouping and segregation.

In relation to your example: Embedding sections in a single is kind of possible now, since you can nest entries in a Matrix block of "element index" view mode! So, for the "Meet the Team" example you could embed the team members in a Matrix field of the parent page.

I'm not sure that's _the_ perfect solve for all cases though. For example, authors would want to have quick/easy access to the entries within each section, ideally without switching context to each parent entry. That's one reason that the recent changes to the asset indexes have raised some concerns: Often you have to go in and out of folders instead of having a tree structure view. It's an interesting UI problem, but I don't want to hijack your discussion any further 🙃
